models/migrations/v226.go

```diff
@@ -62,7 +63,7 @@ func addHeaderAuthorizationEncryptedColWebhook(x *xorm.Engine) error {
 	hooks := make([]*Webhook, 0, 50)
 	for {
 		err := x.Where("type=?", "matrix").OrderBy("id").Limit(50, start).Find(&hooks)
-		if err != nil {
+		if err != nil || len(hooks) == 0 {
```

Comment:
if len(hooks) == 0 {
    break
   }
   
   you can not return here.

Reply:
you are right. I added a `break` as well for `hookTasks`.